pypeit/scripts/chk_tilts.py

```diff
@@ -0,0 +1,45 @@
+"""
+This script displays the Trace image and the traces in an RC Ginga window.
+
+.. include common links, assuming primary doc root is up one directory
+.. include:: ../include/links.rst
+"""
+
+import numpy as np
+from pypeit.scripts import scriptbase
+from IPython import embed
+
+
+class ChkTilts(scriptbase.ScriptBase):
+
+    @classmethod
+    def get_parser(cls, width=None):
+        parser = super().get_parser(description='Display Tiltimg image and 2D fitted tilts in Ginga viewer '
+                                                'or Matplotlib window',
+                                    width=width)
+
+        parser.add_argument('file', type=str, default=None,
+                            help='PypeIt Tilts file [e.g. Tilt_A_1_01.fits]')
+        parser.add_argument('--tiltimg', type=str, default=None,
```

Comment:
Not necessarily a question for this PR, but can we just add the Tiltimg path to the Tilt_A_1_01.fits header, and or add the Tiltimg to that data model. Then we would not need this extra argument (although it might take more space if we duplicate the image storage).

Reply:
Added the Tiltimg file name in the Tilts data model.